newIDE/app/src/ObjectsList/index.js

```diff
@@ -675,6 +683,9 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
           {
             label: i18n._(t`Rename`),
             click: () => editName(objectWithContext),
+            accelerator: getShortcutDisplayName(
+              preferences.values.userShortcutMap['SCENE_OBJECT_RENAME'] || 'F2'
```

Comment:
We should probably NOT have a default value. The shortcut map should be the source of truth and not accelerator, apart from the non customisable one, should live in the code. Otherwise, what happen if the shortcut is not set? F2 would be displayed and would not work. That would be a bug.

Reply:
I agree. FYI I only did that because I found the same thing in `EventSheet/index.js`.

I will remove it.